sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -207,6 +229,11 @@ class PathProperties(DictMixin):
     :ivar int content_length: The size of file if the path is a file.
     :ivar datetime creation_time: The creation time of the file/directory.
     :ivar datetime expiry_time: The expiry time of the file/directory.
+    :ivar str encryption_scope:
```

Comment:
So, `PathProperties` are only used in the `get_paths` API which I didn't see you aadd a test for? Could you add a test for `get_paths` with a file system created with an encryption scope and a at least one directory and file added and then just for this property on each result?

Reply:
Good catch, will add the necessary test for this!